src/SongList.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { useHandleFetchAndLoad } from "./useHandleFetchAndLoad";
+
+export const Songlist = (props: { playlistId: string }): React.ReactElement => {
```

Comment:
We should not redefine what the functional type component is. Let's use the predefined type (easier to read & more correct). Another small nit is I prefer if we don't inline styles like this for functional components, though that's more of a stylistic choice. 

```

type SonglistProps = {
playlistId: string 
}

export const Songlist: React.FC<SonglistProps> = (props) => {
```